tests/Aspire.Azure.Messaging.EventHubs.Tests/ConformanceTestsBase.cs

```diff
@@ -60,4 +66,32 @@ protected static RemoteInvokeOptions EnableTracingForAzureSdk()
      {
          RuntimeConfigurationOptions = { { "Azure.Experimental.EnableActivitySource", true } }
      };
+
+    [ConditionalFact]
+    public void HealthChecksClientsAreReused()
+    {
+        SkipIfHealthChecksAreNotSupported();
```

Comment:
Shouldn't HealthChecks always be supported for EventHubs now? Why is this check needed?

Reply:
I copied it from the base component class test. I assumed it didn't hurt. And it won't break if for any reason the heathcheck is not supported in a new or updated client. I'll remove it in main.